config/resources/besluit-domain.lisp

```diff
@@ -207,7 +206,16 @@
              (concept                   :via      ,(s-prefix "dct:type")
                                         :as "kind")
              (meeting                   :via      ,(s-prefix "dct:isPartOf")
-                                        :as "main-meeting"))
+                                        :as "main-meeting")
+             (internal-decision-publication-activity :via  ,(s-prefix "ext:internalDecisionPublicationActivityUsed")
+                                        ; optional:  present in new meetings since KAS-3431 / optional in older meetings
+                                        :inverse t
+                                        :as "internal-decision-publication-activity")
+             (internal-document-publication-activity :via  ,(s-prefix "ext:internalDocumentPublicationActivityUsed")
+                                        ; optional: present in new meetings since KAS-3431 / optional in older meetings
+                                        :inverse t
```

Comment:
as said earlier, maybe they shouldn't be optional? just have one without any dates filled in.
That way, they can always be set later and is more consistent to use them in the frontend (no `null or undefined`)